### PR TITLE
Finalize Create Sighting screen

### PR DESCRIPTION
**Pull request by adzialocha**
[Screen_recording_20231011_232359.webm](https://github.com/p2panda/meli/assets/1822473/350f8923-b05c-4cab-8e0f-ebe427e7775b)

### CODE REVIEW

File: packages/app/lib/ui/widgets/image_carousel.dart [hunk 1]
```diff
@@ -1,81 +1,145 @@
...
+    double width = double.infinity;
+
+    if (this.imagePath.contains('http')) {
+      return Image.network(this.imagePath, fit: fit, width: width);
+    } else if (this.imagePath.startsWith('/')) {
+      return Image.file(File(this.imagePath), fit: fit, width: width);
+    } else {
+      return Image.asset(this.imagePath, fit: fit, width: width);
+    }
+  }
```
Comment (sandreae) [inline packages/app/lib/ui/widgets/image_carousel.dart:123]: I wonder if users will be bothered that their images get cropped when taken portrait? I don't see a great way around it unless we make changes to the design (square images?). I could imagine important bits of bees getting cropped off if users aren't careful.

What are your thoughts? Could open a separate issue if we wanna discus more as it doesn't need to hold up this pr.

**Review by sandreae — APPROVED** [review]
I had one comment about image cropping, this can be discussed in a separate issue if we want though. Otherwise all good, really looking great!

File: packages/app/lib/ui/widgets/image_carousel.dart (hunk 1 above)
Comment (adzialocha) [reply packages/app/lib/ui/widgets/image_carousel.dart:123]: I think we should have a way to look at images fullscreen and not in this little widget. Can imagine that for proper identification that's too small.

Comment (adzialocha) [reply packages/app/lib/ui/widgets/image_carousel.dart:123]: I've made a ticket!

**Pull request merged**